### DISCUSSION TITLE
Keyboard shortcuts

### DISCUSSION THREAD

**Discussion by aviggiano** [solodit/feedback#16] [New Feature Requests]
As a Solodit power user, I would like to use Vim-like shortcuts to read issues faster.

I would like tu use `j` to go to the next issue, `k` to go to the previous issue, `/` to search, so on and so forth.

For example, Gmail has that:

![image](https://github.com/solodit/feedback/assets/3029017/4d5367c9-43af-4a71-b871-8c85cc740724)

**Comment by Grimxjoke** [comment]
Could save so much time. Great Idea
